scio-core/src/main/scala/com/spotify/scio/metrics/package.scala

```diff
@@ -36,11 +36,9 @@ package object metrics {
    */
   case class Metrics(version: String,
```

Comment:
Btw - since you are changes stuff around here, could you please make all those case classes `final`?